cdap-common/bin/cdap

```diff
@@ -83,7 +83,8 @@ case ${1} in
   upgrade) shift; cdap_upgrade_tool ${@}; __ret=${?} ;;
   version) shift; cdap_version_command ${@}; __ret=${?} ;;
   run) shift; cdap_run_class ${@}; __ret=${?} ;;
-  sdk) shift; cdap_sdk ${@}; __ret=${?} ;;
+  sdk) shift; echo "[WARN] sdk commands are deprecated, please use sandbox commands instead."; cdap_sdk ${@}; __ret=${?} ;;
```

Comment:
should we do ``>&2 echo [WARN] ...`` so that the output goes to stderr? That way, if someone has automation around the CLI that expects things on stdout, that won't fail.